webcompat/issues.py

```diff
@@ -43,6 +43,31 @@
 REJECTED_BODY = '''<p>The content of this issue doesn't meet our
 <a href="https://webcompat.com/terms#acceptable-use">acceptable use</a>
 guidelines. Its original content has been deleted.</p>'''
+AUTOCLOSE_TITLE = 'Issue closed.'
+AUTOCLOSE_BODY = '''<p>Thanks for the report. We have closed this issue
+automatically as we suspect it is invalid. If we made a mistake, please
+file a new issue and try to provide more context or file a report
+<a href="https://github.com/webcompat/web-bugs/issues/new/choose">
+via our GitHub repository</a>.</p>'''
+
+TEMPLATE_CONFIG = {
+    'rejected': {
+        'title': REJECTED_TITLE,
+        'body': REJECTED_BODY
+    },
+    'invalid': {
+        'title': '',
+        'body': INVALID_BODY
+    },
+    'incomplete': {
+        'title': '',
+        'body': INCOMPLETE_BODY
+    },
+    'autoclosed': {
+        'title': AUTOCLOSE_TITLE,
+        'body': AUTOCLOSE_BODY
+    }
+}
```

Comment:
possibly here ongoing could be part of this structure too. 
```suggestion
    'autoclosed': {
        'title': AUTOCLOSE_TITLE,
        'body': AUTOCLOSE_BODY
    }
    'ongoing': {
        'title': ONGOING_TITLE,
        'body': ONGOING_BODY
    }
}
```

Then the `if` in `moderation_template` becomes unecessary. it just extracts the right case.

Reply:
Yeah, except when a key that is not in the config is passed, it defaults to ongoing too. 

I just saw this test https://github.com/webcompat/webcompat.com/blob/master/tests/unit/test_issues.py#L146 and wasn't sure if I should go this path :) It's probably safe though, just need to check all references of the function

Reply:
moved it to the config, added an exception for the wrong key and adjusted the test